src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import socialMedia from '../../assets/socialMedia.svg';
+import logo from '../../assets/logo.svg';
+
+const Footer: React.FC = () => {
+    return (
+        <footer className="bg-primary text-white pt-[51px] max-sm:px-4 sm:max-md:px-8 max-sm:pb-[62px] sm:max-md:pb-[67px] lg:pt-[92px] lg:pb-[54px] font-Roboto">
+            <div>
+                <p className="text-sm text-left text-grey font-light md:ml-[156px]">Powered by:</p>
+                <img className="w-[302px] h-[39px] md:w-[377px] md:h-[49px] mt-1 md:ml-[151px]" src={logo} alt="Logo" />
+
+                <p className="text-sm md:text-lg mt-[67px] lg:mt-[65px] md:ml-[156px] md:mr-[152px] text-start font-normal">
+                    Instant Risk Coverage is a trademark of Instant Risk
+                    Coverage Inc. Refer to your policy for the most detailed and
+                    accurate information about your coverage and terms of
+                    insurance. Your policy, which serves as your insurance
+                    contract, will always prevail if there's ever a conflict
+                    with the information found on this site.
+                </p>
+                <div className="flex justify-start md:justify-end mt-[47.5px] md:mr-[130px]">
+                    <a href="">
+                        <img className="w-[128px] h-[33.49px]" src={socialMedia} alt="Social Media" />
+                    </a>
```

Comment:
As per design, there are 3 social media links. So technically there should be 3 anchor tags each pointing to specific social media platform